django/contrib/admin/options.py

```diff
@@ -979,6 +975,22 @@ def log_deletion(self, request, obj, object_repr):
             action_flag=DELETION,
         )
 
+    def log_deletions(self, request, queryset, message=""):
+        """
+        Log that multiple objects will be deleted. Note that this method must
+        be called before the deletion of multiple objects.
+
+        The default implementation creates some LogEntry objects.
+        """
+        from django.contrib.admin.models import DELETION, LogEntry
+
+        return LogEntry.objects.log_actions(
+            user_id=request.user.pk,
+            queryset=queryset,
+            action_flag=DELETION,
+            change_message=message,
+        )
```

Comment:
`log_deletion` should also be deprecated.

`log_deletions` must be backward compatible, i.e. call `log_deletion` when it's overridden in subclasses.

Reply:
Same would also be required for the` log_action` and the `log_actions` method in the `LogEntryManager` class right?